internal/healthcheck/target_json.go

```diff
@@ -59,5 +60,9 @@ func (t *TargetJSON) httpRequestConfig() *HttpRequestConfig {
 		}
 	}
 
+	if t.HttpRequestConfig.ExpectedResponseCode == 0 {
+		t.HttpRequestConfig.ExpectedResponseCode = http.StatusOK
```

Comment:
you need to pass a defualt to line 59 as well

Reply:
@aizatto ok, i missed it